modin/core/dataframe/base/partitioning/axis_partition.py

```diff
@@ -19,7 +19,9 @@
 from modin.logging import ClassLogger
 
 
-class BaseDataframeAxisPartition(ABC, ClassLogger):  # pragma: no cover
+class BaseDataframeAxisPartition(
+    ABC, ClassLogger, modin_layer="AXIS-PARTITION"
```

Comment:
```suggestion
    ABC, ClassLogger, modin_layer="VIRTUAL-PARTITION"
```
Since we had plans to rename AxisPartition to VirtualPartition, I think it makes sense to use it as modin_layer name.